kuma/wiki/tests/test_views_document.py

```diff
@@ -167,7 +167,7 @@ def test_api_safe(client, section_doc, section_case, if_none_match, method):
                 str(section_doc.current_revision_id))
 
     if method == 'GET':
-        assert response.content == exp_content
+        assert response.content == exp_content.encode('utf-8')
```

Comment:
```suggestion
        assert response.content == exp_content.decode('utf-8')
```

Reply:
I'd expect ``response.content`` to be a UTF-8 encoded binary string, and ``exp_content`` to be a binary string in Python 2 and a unicode string in Python 3. I think you need to change line 158 to:

```
exp_content = b''
```
or, better yet, get rid of the variable and change this line to:

```suggestion
        assert response.content == b''
```